tests/integration/targets/aci_epg_to_domain/tasks/main.yml

```diff
@@ -208,6 +208,163 @@
     enhanced_lag_policy: enhanced
   register: enhanced_lag_policy
 
+- name: bind vmm domain to epg - vmm_uplink
```

Comment:
Since all the below test cases are using the condition  when: version.current.0.topSystem.attributes.version is version('4.2', '>=')

we can put all these in a block (block:)
for example:  in       tests/integration/targets/aci_cloud_external_epg/tasks/main.yml

Reply:
ok makes sense will change.